package/MDAnalysis/core/universe.py

```diff
@@ -671,8 +672,37 @@ def trajectory(self, value):
         del self._trajectory  # guarantees that files are closed (?)
         self._trajectory = value
 
-    def add_TopologyAttr(self, topologyattr):
-        """Add a new topology attribute."""
+    def add_TopologyAttr(self, topologyattr, values=None):
+        """Add a new topology attribute to the Universe
+
+        Adding a TopologyAttribute to the Universe makes it available to
+        all AtomGroups etc throughout the Universe.
+
+        Parameters
+        ----------
+        topologyattr : TopologyAttr or string
+          Either a MDAnalysis TopologyAttr object or the name of a possible
+          topology attribute.
+        values : np.ndarray, optional
+          If initiating an attribute from a string, the initial values to
+          use.  If not supplied, the new TopologyAttribute will have empty
+          or zero values.
```

Comment:
*Please* add a canonical example such as adding B-factors (tempFactor) https://github.com/MDAnalysis/mdanalysis/issues/1359#issuecomment-320382240 . It will make our life SO MUCH easier when we can point users to some place in the docs.

Reply:
done